backend/apps/sandbox/serializers.py

```
@@ -158,4 +158,32 @@ class Meta:
+
+    def create(self, validated_data):
+        validated_data['user'] = self.context['request'].user
+        return super().create(validated_data)
+
+
+from .models import CollabSession, CollabSessionLog
+
+class CollabSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CollabSession
+        fields = ["id", "project", "allowed_users", "created_at", "is_active"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)